tmp/build/patch-charts.sh

```diff
@@ -258,17 +258,28 @@ function patchTracingtemplate() {
   # update jaeger image hub
   if [[ "${COMMUNITY,,}" == "true" ]]; then
     sed -i -e 's+hub: docker.io/jaegertracing+hub: jaegertracing+g' \
-           -e 's+tag: 1.9+tag: 1.11+g' ${HELM_DIR}/istio/charts/tracing/values.yaml
+           -e 's+tag: 1.9+tag: 1.12+g' ${HELM_DIR}/istio/charts/tracing/values.yaml
   else
     sed -i -e 's+hub: docker.io/jaegertracing+hub: distributed-tracing-tech-preview+g' \
-           -e 's+tag: 1.9+tag: 1.11.0+g' ${HELM_DIR}/istio/charts/tracing/values.yaml
+           -e 's+tag: 1.9+tag: 1.12.0+g' ${HELM_DIR}/istio/charts/tracing/values.yaml
   fi
 
+  # add default template
+  sed -i -e '/^jaeger:/a\
+\  template: production-elasticsearch' ${HELM_DIR}/istio/charts/tracing/values.yaml
+
   # update jaeger zipkin port name
   sed -i -e '/service:$/,/externalPort:/ {
-    s/name:.*$/name: jaeger-collector-zipkin/
+    s/name:.*$/name: zipkin/
```

Comment:
Curious why the port is being renamed?

Reply:
No idea.  Leftover from early days.  It applies to [this](https://github.com/Maistra/istio/blob/maistra-0.12/install/kubernetes/helm/istio/charts/tracing/values.yaml#L59), which is manifested [here](https://github.com/Maistra/istio/blob/maistra-0.12/install/kubernetes/helm/istio/charts/tracing/templates/service.yaml#L28).  Basically, it just names the port on the service `zipkin`.